src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRulesLoader.java

```diff
@@ -46,7 +46,10 @@ private MappingRulesLoader() {
      *
      * @return the singleton instance of the MappingRulesLoader
      */
-    public static MappingRulesLoader getInstance() {
+    public static MappingRulesLoader getInstance() throws NullPointerException {
```

Comment:
@antonio-pedro99 Should thread safety be a consideration here?

In its current implementation, multiple threads can potentially access the getInstance() method simultaneously and create multiple instances of the MappingRulesLoader class, which contradicts the intention of having a singleton pattern. If this is not a concern, kindly ignore this. 

For more info on the topic check out this article:

https://docs.oracle.com/javase/tutorial/essential/concurrency/sync.html

Reply:
Yeap, you are right!
Thanks for point it out @AlexJuca.